application/src/objects/car.js

```diff
@@ -95,4 +97,32 @@ Car.prototype.applyForces = function()
     );
 };
 
+Car.prototype.fall = function()
+{
+    this.falling = true;
+
+    // Adjust car position slightly to avoid some cases where the falling
+    // effect is shown while not over a drop tile.
+    if (Math.abs(this.body.velocity.x) > Math.abs(this.body.velocity.y)) {
+        this.body.x += (50 * (this.body.velocity.x > 0 ? 1 : -1));
+    }
+    else if (Math.abs(this.body.velocity.y) > 0) {
+        this.body.y += (50 * (this.body.velocity.y > 0 ? 1 : -1));
+    }
```

Comment:
Okay. I think I wouldn't have this problem if `setTileIndexCallback` worked. I assume if it did I would have access to the exact tile that was collided with, whereas now I have access to a P2 physics body that encompasses many tiles, so it's not possible to pinpoint the exact collision spot. In the source `setTileIndexCallback` adds the callback to an array of callbacks on the layer object, but I can't seem to find where it actually does anything with that callback array. I should delve into this further and see if I can figure out what's happening. Maybe it's a bug.